terra-jupyter-hail/Dockerfile

```diff
@@ -1,13 +1,17 @@
 FROM us.gcr.io/broad-dsp-gcr-public/terra-jupyter-python:0.0.8
 USER root
-ENV PIP_USER=false
 
+COPY scripts $JUPYTER_HOME/scripts
```

Comment:
This is a similar pattern to the [terra-jupyter-r](https://github.com/DataBiosphere/terra-docker/tree/master/terra-jupyter-r) Dockerfile, which also adds a custom kernepsec.